osu.Game/Localisation/DownloadButtonStrings.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class DownloadButtonStrings
```

Comment:
Not sure at all about the naming for this class but this should be ok since this will probably get reused for any button doing a download?

Reply:
Well there's this (which is seemingly not replaced in this PR):

https://github.com/ppy/osu/blob/2b0f2a6db9efa93ebd1cfce97ba8bba6674aaec3/osu.Game/Graphics/UserInterface/DownloadButton.cs#L46-L63

But I'm not super sure about having it here rather than in common strings or something.

Reply:
Common may be a better place for these. I can imagine usage outside of the button, as well.